tests/formats/light_client/single_merkle_proof.md

```diff
@@ -5,24 +5,30 @@ generation and verification of merkle proofs based on static data.
 
 ## Test case format
 
-### `state.ssz_snappy`
+### `meta.yaml`
 
-An SSZ-snappy encoded `BeaconState` object from which other data is generated.
+```yaml
+object_class: string  -- 'BeaconState'
```

Comment:
Instead of "object_class" should it be called "type_name" or "container_name"? More consistent with the notation here https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#custom-types

Reply:
Moved it to folder name instead, avoids the variable name :)